panel/src/components/Collection/Items.vue

```diff
@@ -38,7 +38,8 @@
 					:layout="layout"
 					:link="link ? item.link : false"
 					:selecting="selecting"
-					:selectable="item.selectable"
+					:selectable="item.selectable !== false"
```

Comment:
`Explanation:` I think items should be selectable by default and only disabled, when explicitly marked as not selectable.